.cirrus.yml

```diff
@@ -0,0 +1,29 @@
+alpine_task:
+        container:
+                image: alpine:latest
+        install_script: apk add autoconf automake g++ gcc libusb-dev libtool linux-headers eudev-dev make musl-dev
+        script:
+                - ./bootstrap
+                - ./configure || { cat config.log; exit 1; }
```

Comment:
Why does this configure statement have the statement behind but not the others?

Reply:
Oh, I probably added that while trying to track down some problem building on alpine, and did not run into a problem on the other OSes that required more details. It's probably a good idea for all cases and I'll update the PR.